scripts/mkcloudcloud/deploy.sh

```diff
@@ -57,6 +61,15 @@ export want_tempest_proposal=1
 ./mkcloud "\$@"
 EOF
 chmod a+x cloud9
+
+if [ ! -f ~/.ssh/id_rsa.pub ]; then
+  ssh-keygen -t rsa -f /root/.ssh/id_rsa -N ""
+  cat ~/.ssh/id_rsa.pub >> ~/.ssh/authorized_keys
+  chmod og-wx ~/.ssh/authorized_keys
```

Comment:
This chmod would normally be a no-op, because the default umask 22 drops write bits for group+others

Reply:
changed that to 600